scripts/rebase.sh

```diff
@@ -241,6 +229,13 @@ regenerate_openapi() {
 }
 
 
+# Returns the list of release image names from a release_${arch}.go file
```

Comment:
What do you think about converting the image lists to YAML (or some other text format) that we put in the assets directory? Then this script could use yq, for example.